packages/server-renderer/src/render.ts

```diff
@@ -144,7 +144,10 @@ function renderComponentSubTree(
     // perf: enable caching of computed getters during render
     // since there cannot be state mutations during render.
     for (const e of instance.scope.effects) {
-      if (e.computed) e.computed._cacheable = true
+      if (e.computed) {
+        e.computed._dirty = true
```

Comment:
Make sure that during `ssrRender()`, the computed is executed once and then cached.